Xamarin.Forms.Platform.GTK/Controls/ScrolledTextView.cs

```diff
@@ -18,14 +19,25 @@ public ScrolledTextView()
                 WrapMode = WrapMode.WordChar
             };
 
+            _textView.Buffer.InsertText += InsertText;
             Add(_textView);
         }
 
         public TextView TextView => _textView;
 
+        public void SetMaxLength(int maxLength)
```

Comment:
Any reason for an empty method here?

Reply:
Simply forgot to tie it together here.. Did it now!